docs/Install-with-AC-Dashboard.md

```diff
@@ -0,0 +1,179 @@
+## Introduction
+
+The AzerothCore bash dashboard is a collection of scripts that help with the installation and maintenance of an AzerothCore server.
+It allows to easily install, update and execute AzerothCore on your machine in a tremendously small amount of steps.
+
+Installing a development or production private server has never been so easy before. 
+If you need any help just [ask a question](https://www.azerothcore.org/wiki/How-to-ask-for-help).
+
+## Requirements
+
+You need to have [git](https://git-scm.com/) installed in your machine. 
+No other software is required to be installed manually.
+
+- debian/ubuntu-based: `apt update && apt install git`
+- macOS: `brew install git`
+
+
+### Notes
+- For macOS users: install and use the latest version of bash to run the commands of the dashboard (`brew install bash`)
+- For Windows users: the commands need to be executed within the "git bash" shell or a bash-compatible shell such as WSL, cygwin etc..
+  However, git bash is suggested because is preinstalled with git for windows (one of our requirements)
+  
+
+## Setup
+
+### Get the AC sources
+
+
+```
+git clone https://github.com/azerothcore/azerothcore-wotlk.git; cd azerothcore-wotlk`
+```
+
+### Configuration
+
+There is a [conf/dist/config.sh](https://github.com/azerothcore/azerothcore-wotlk/blob/master/conf/dist/config.sh) 
+file containing a default configuration. Have a look at it. 
+Most of the configuration defaults will probably work for your case, 
+but you can copy it under `conf/config.sh` and change values as you please.
+
+
+### Install all AC dependencies
+
+```
+./acore.sh install-deps
+```
+
+### Build everything from scratch
+
+```
+./acore.sh compiler all
+```
+
+### Set-up the database
+
+- Either connect to your MySQL database and manually create the `acore` MySQL user:
+
+```
+DROP USER IF EXISTS 'acore'@'localhost';
+DROP USER IF EXISTS 'acore'@'127.0.0.1';
+CREATE USER 'acore'@'localhost' IDENTIFIED BY 'acore';
+CREATE USER 'acore'@'127.0.0.1' IDENTIFIED BY 'acore';
```

Comment:
We should suggest that this is only an example and to make sure to change the acore password to something more secure since acore is a well known default use in ac.

Reply:
or just limit the access of the `acore` user to localhost... I took it for granted but it's better to specify it, yes

Reply:
@locus313 but it's bound to `localhost`, so it cannot be accessed from outside. Do you think it's still needed?